### PR TITLE
Initial package deployment

### PR DESCRIPTION
**Pull request by anilnatha**
This PR contains the initial version of the Jupyter Extension that provides users the ability to hide the Che sidebar from the Jupyter environment.

### CODE REVIEW

File: package.json
Comment (marjo-luc) [inline package.json:1]: You may want to consider publishing this extension to the @maap-jupyterlab namespace, where we keep all of our MAAP npm jupyter extensions. To do that, you would change line 2

from:
`"name": "che-sidebar-visibility-jupyter-extension",`

to:
`"name": "@maap-jupyterlab/che-sidebar-visibility-jupyter-extension",`

**Review by marjo-luc — COMMENTED** [review]
Looks good. I'll approve the PR if the package namespace is updated or justification for not publishing it to that namespace is provided.

File: package.json
Comment (anilnatha) [reply package.json:1]: Thank you for reviewing and catching this. Yes, I intended to publish thie package to the @maap-jupyterlab namespace. I'll make the change later today or tomorrow and report back. Have a meeting to attend at the moment.

Comment (anilnatha) [reply package.json:1]: @marjo-luc I've updated the namespace. Thanks again for your guidance!

**Review by marjo-luc — APPROVED** [review]

**Pull request merged**